extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -55,13 +55,30 @@ export class AmpStoryRequestService {
 
   /**
    * Retrieves the publisher share providers.
-   * Has to be called through `loadShareConfig`.
+   * @param  {!Element} shareConfigEl
    * @return {(!Promise<!JsonObject>|!Promise<null>)}
    */
-  loadShareConfigImpl_() {
-    const shareConfigEl = this.storyElement_.querySelector(
-      'amp-story-social-share, amp-story-bookend'
-    );
+  getInlineConfig(shareConfigEl) {
+    try {
+      return Promise.resolve(getChildJsonConfig(shareConfigEl));
+    } catch (err) {
+      return Promise.reject(err);
+    }
+  }
+
+  /**
+   * Retrieves the publisher share providers.
+   * @param  {?Element} element
+   * @return {(!Promise<!JsonObject>|!Promise<null>)}
+   * @public
```

Comment:
You can remove this line because only non-public fields need to be annotated with the proper visibility annotation (i.e., `@private`, `@protected`, `@package`)